pkg/api/validate/format_regexps.go

```diff
@@ -16,4 +16,8 @@ var (
 		`([a-z][-a-z0-9]{0,61}[a-z0-9])` +
 		`(\.([a-z0-9]|[a-z0-9][-a-z0-9]{0,61}[a-z0-9]))*` +
 		`$`)
+	RxDomainNameRFC1123 = regexp.MustCompile(`^` +
```

Comment:
It is a regex we previously had for validating domains and I'm actually not 100% sure that this regex reflects RFC1123. 

Any suggestions for better name are welcome.